### DISCUSSION TITLE
Trying to test 2.4

### DISCUSSION THREAD

**Discussion by Wilk4013** [Security-Onion-Solutions/securityonion#10689] [2.4] [answered]
Hello,
I am trying to test 2.4. I dowloaded the iso and built a unit with the specs of 8 core, 16 gig ram and 200 SSD storage. I am seeing traffic on the monitor port but no data in the SOC interface. In 2.3 I can look in Grafana for clues but I don't see the option in 2.4. Can someone give me some clues on where to look to try and solve this issue?

**Comment by InfosecGoon** [comment, marked as answer]
In 2.4, Grafana has been replaced with Influx - the menu option is in roughly the same place in SOC.

Are there any clues in the logs for logstash or elasticsearch under /opt/so/logs?

**Reply by Wilk4013** [reply]
Hey man it looks like a layer 8 error somewhere. Rebuilt and now things are good. Thanks Goon!!!

**Comment by Wilk4013** [comment]
Matthew
I did see the Influxdb dashboard, thank you. I reverted back to 2.3.60 but
I will try again with 2.4.3 and check the logs you suggested and
report back. Thank you for your help
